ui/src/app/pages/admin/workspace/basic-information.tsx

```diff
@@ -100,51 +102,59 @@ export const BasicInformation = ({
           {new Date(workspace.lastModifiedTime).toDateString()}
         </WorkspaceInfoField>
         <WorkspaceInfoField labelText='Workspace Published'>
-          <Select
-            key={featuredCategory || 'placeholder'}
-            value={featuredCategory}
-            placeholder='Select a category...'
-            isDisabled={featuredCategoryLoading}
-            options={FeaturedWorkspaceCategoryOptions}
-            onChange={(v) => setFeaturedCategory(v)}
-          />
-          <TooltipTrigger
-            disabled={!publishingDisabled}
-            content={getWorkspacePublishTooltip()}
-          >
-            <Button
-              type='primary'
-              disabled={publishingDisabled}
-              onClick={() => {
-                setFeaturedCategoryLoading(true);
-                workspaceAdminApi()
-                  .publishWorkspaceViaDB(workspace.namespace, {
-                    category: featuredCategory,
-                  })
-                  .then(async () => {
-                    await reload();
-                  });
-              }}
-            >
-              Publish
-            </Button>
-          </TooltipTrigger>
-          <Button
-            type='secondaryOutline'
-            disabled={featuredCategoryLoading || !workspace.featuredCategory}
-            onClick={() => {
-              setFeaturedCategory(null);
-              setFeaturedCategoryLoading(true);
-              workspaceAdminApi()
-                .unpublishWorkspaceViaDB(workspace.namespace)
-                .then(async () => {
-                  await reload();
-                });
-            }}
-          >
-            Unpublish
-          </Button>
-          {featuredCategoryLoading && <Spinner size={36} />}
+          {!enablePublishedWorkspacesViaDb &&
```

Comment:
Previous UI when ff is disabled.

Reply:
optional: I generally prefer this pattern instead
```
{enablePublishedWorkspacesViaDb ? (true-case UI) : (false-case UI)}
```

Reply:
Yeah, I generally prefer ternary too, but I was thinking that this is so many lines, the different sections of the ternary might not be as discernible. I am happy to swap it over though.